docs/sources/reference/run.md

```diff
@@ -873,18 +873,26 @@ this driver.
 Default logging driver for Docker. Writes JSON messages to file. `docker logs`
 command is available only for this logging driver
 
+The following logging options are supported for this logging driver: [none]
+
 #### Logging driver: syslog
 
 Syslog logging driver for Docker. Writes log messages to syslog. `docker logs`
 command is not available for this logging driver
 
+The following logging options are supported for this logging driver:
+
+    --log-opt address=[tcp|udp]://host:port
+
+`address` specifies the remote syslog server address where the driver connects to.
+If not specified it defaults to the local unix socket of the running system. If
```

Comment:
I needed to specify a custom socket (not `/dev/log`) for docker's syslog driver to use, and made a fork pre-1.6 release. I also used `Dial`; the line looks like:

    log, err := syslog.Dial("unixgram" , syslog_socket , syslog.LOG_DAEMON, fmt.Sprintf("%s/%s", path.Base(os.Args[0]), tag))

where a `syslog_socket := "/dev/log"` gets you the old default syslog behavior.